src/components/ContinuationIn/ContinuationAuthorization.vue

```diff
@@ -95,47 +97,34 @@
 
     <!-- file upload components -->
     <v-row
+      v-for="(num, index) in numUploads"
+      :key="authorization.files[index]?.fileKey"
```

Comment:
This (and below) is a reversion of the previous code, which shows additional file inputs dynamically to upload additional files (up to 5).